diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/android/AndroidLibraryExtension.kt b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/android/AndroidLibraryExtension.kt
index 9bfae8d..69c35d0 100644
--- a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/android/AndroidLibraryExtension.kt
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/android/AndroidLibraryExtension.kt
@@ -6,8 +6,9 @@ internal fun LibraryExtension.applySDKVersions(compileSdk: Int, minSdk: Int, tar
   this.compileSdk = compileSdk
   defaultConfig {
     this@defaultConfig.minSdk = minSdk
-    this@defaultConfig.targetSdk = targetSdk
   }
+  testOptions.targetSdk = targetSdk
+  lint.targetSdk = targetSdk
 }
 
 internal fun LibraryExtension.applyLinterOptions() {
