diff --git a/node_modules/expo-font/android/build.gradle b/node_modules/expo-font/android/build.gradle
index 7de989e..3636e01 100644
--- a/node_modules/expo-font/android/build.gradle
+++ b/node_modules/expo-font/android/build.gradle
@@ -9,6 +9,7 @@ def expoModulesCorePlugin = new File(project(":expo-modules-core").projectDir.ab
 if (expoModulesCorePlugin.exists()) {
   apply from: expoModulesCorePlugin
   applyKotlinExpoModulesCorePlugin()
+  useDefaultAndroidSdkVersions()
   // Remove this check, but keep the contents after SDK49 support is dropped
   if (safeExtGet("expoProvidesDefaultConfig", false)) {
     useExpoPublishing()
@@ -43,17 +44,17 @@ buildscript {
 // Remove this if and it's contents, when support for SDK49 is dropped
 if (!safeExtGet("expoProvidesDefaultConfig", false)) {
   afterEvaluate {
-    publishing {
-      publications {
-        release(MavenPublication) {
-          from components.release
+      publishing {
+        publications {
+          release(MavenPublication) {
+            from components["release"]
+          }
         }
-      }
-      repositories {
-        maven {
-          url = mavenLocal().url
+        repositories {
+          maven {
+            url = mavenLocal().url
+          }
         }
-      }
     }
   }
 }
@@ -61,11 +62,11 @@ if (!safeExtGet("expoProvidesDefaultConfig", false)) {
 android {
   // Remove this if and it's contents, when support for SDK49 is dropped
   if (!safeExtGet("expoProvidesDefaultConfig", false)) {
-    compileSdkVersion safeExtGet("compileSdkVersion", 34)
+      compileSdkVersion safeExtGet("compileSdkVersion", 35)
 
     defaultConfig {
-      minSdkVersion safeExtGet("minSdkVersion", 23)
-      targetSdkVersion safeExtGet("targetSdkVersion", 34)
+        minSdkVersion safeExtGet("minSdkVersion", 24)
+        targetSdkVersion safeExtGet("targetSdkVersion", 35)
     }
 
     publishing {
